### PR TITLE
metrics-webapp/first version

### PR DESCRIPTION
**Pull request by shahadat3669**
## React capstone project - Metrics webapp
Group 7 (2)
This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that make by the use of React and Redux.
### Requirement:
- Choose [REST COUNTRIES API](https://restcountries.com/#api-endpoints-v3-region) which allows getting numeric values based on region parameters. 
- Follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one main color for your website).
  - Typography: font face, size, and weight.
  - Layout: composition and space between elements.
  _Note: There were two fonts Lato and Gill sans in the [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). But [Gill sans](https://www.myfonts.com/collections/gill-sans-font-monotype-imaging) is a paid font, so I only use Lato in this project._
- Home page:
  - When the page loads, the web app shows the list of categories that could be filtered out by region parameter
  - Along with the region name, Display some numeric values per country that come from the API.
  - When the user clicks (or taps) on a Country, the application navigates to the details page.
- Details page
  - On the details page, the web app retrieves data from the API to show detailed data in the country.
  - When the user clicks on the "Back" button (<), the user navigates to the home page.
- Testing requirements
  - Create unit tests for pure functions in Redux code.
  - Create integration tests for applications using the React Testing Library.
- Technical requirements
  - The project is a single-page application (SPA) built with React and Redux.
  - The data retrieved from the API is stored in the Redux store.
  - Filter the data that retrieve from the API using a Filter stateless component.
  - Every page (the main page and the pages for each item) have a unique route within the SPA.
  - The project is deployed and accessible online.

### General Requirements:
- There are [no linter errors](https://github.com/microverseinc/linters-config).
- Used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Follow the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Follow the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

File: src/utils/regions.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const regions = [
+  { id: 0, name: 'europe', img: 'europe.svg' },
+  { id: 1, name: 'asia', img: 'asia.svg' },
+  { id: 2, name: 'africa', img: 'africa.svg' },
+  { id: 4, name: 'oceania', img: 'oceania.svg' },
+  { id: 5, name: 'americas', img: 'americas.svg' },
+];
+
```
Comment (hiromataba) [inline src/utils/regions.js:8]: - [x] This is very good I really appreciate 👍

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @shahadat3669     :wave: :wave: 

200w  

## Highlights 🔥 🚀  🍾 

- Nice commit history 👍 
- Nice PR description 🚀 
- Nice Design including the pdf doc presentation 👍 
- You really did a great job ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/utils/regions.js (hunk 1 above)
Comment (shahadat3669) [reply src/utils/regions.js:8]: @hiromataba  Thanks a lot.

Comment (hiromataba) [reply src/utils/regions.js:8]: You are welcome @shahadat3669 🙏

**Pull request merged**